extension/app/src/components/input_bar/InputBarContainer.tsx

```diff
@@ -81,6 +72,22 @@ export const InputBarContainer = ({
     "whitespace-pre-wrap font-normal"
   );
 
+  const onClick = async () => {
+    const jsonContent = editorService.getTextAndMentions();
+    onEnterKeyDown(editorService.isEmpty(), jsonContent, () => {
+      editorService.clearEditor();
+    });
+  };
+
+  const SendAction = {
+    label: "Send",
+    onClick,
+  };
+  const SendWithContentAction = {
+    label: "Send with tab content",
```

Comment:
UXWriter assistant suggest: 

_Basic: "Send message"
With tab: "Send message + tab"_

WDWT?

Reply:
cc @pinotalexandre

Reply:
Let's stick with the original one :)